_schedule/talks/2023-10-17-17-15-t0-an-approach-to-lightweight-tenancy-management-using-django-rest-framework.md

```diff
@@ -30,7 +30,7 @@ track: t0
 Over the last few years, I have run into the same multitenancy use case across different projects. This scenario is a “lightweight” multitenancy use case, where we have a tenant model and tenants are instances of this model; all tenants share the same database, schema, and application instance. Resources belong to a single tenant, but users can belong to multiple tenants. Almost all api routes need to be nested under the tenant id, with urls of the form `api/tenants/tenant-id/some-resource`. The challenges we faced were how to effectively nest our API urls and how to consistently restrict access to resources, so that users could only access those resources that belong to tenants that the user has permission to access. 
```

Comment:
FYI... you've used both multitenancy and multi-tenancy and api and API.

Reply:
Oof, you're right! I'll open a new PR with the changes, thanks